app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :ordered, -> {order created_at: :desc}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
```
khi chúng ta viết validate như này có sự khác biệt gì với validate của em k?

Reply:
theo em hiểu là nếu để là user, thì chỉ cần user đó xuất hiện, còn với user_id thì cần có cả xác thực là đúng id của user đó ạ